python/ctsm/site_and_regional/run_tower.py

```diff
@@ -167,9 +167,40 @@ def parse_neon_listing(listing_file, valid_neon_sites):
     return available_list
 
 
+def check_plumber_data(valid_plumber_sites):
+    """
+    A function to find plumber sites with the dates
+    where data is available.
+
+    Returns:
+        available_list :
+            list of plumber_site objects that is found
+    """
+
+    available_list = []
+
+    for site_name in valid_plumber_sites:
+
+        # start_year and end_year are set in shell commands, so these get overwritten
+        start_year = 2018
```

Comment:
Is this needed?  I'll let you investigate.

Reply:
I changed these start and and years to be very clearly dummy years. I think this is necessary because the PlumberSite class must be implemented with a start/end time. However, this is changed later in the shell commands and is not a known parameter at the time of the object instantiation.